pkg/rtc/participant.go

```diff
@@ -2055,3 +2057,16 @@ func (p *ParticipantImpl) GetCachedDownTrack(trackID livekit.TrackID) (*webrtc.R
 
 	return nil, sfu.ForwarderState{}
 }
+
+func (p *ParticipantImpl) handleNegotiationFailed() {
+	p.params.Logger.Infow("negotiation failed, notify client do full reconnect")
```

Comment:
not close peerconnection here, since we have pc.state check, pc might still connect and negotiated media track is working, so not close it.